src/drawer.canvasentry.js

```diff
@@ -168,17 +170,25 @@ export default class CanvasEntry {
      * @since 5.3.0
      * @param {CanvasRenderingContext2D} ctx Rendering context of target canvas
      * @param {string|string[]} color Fill color for the wave canvas, or an array of colors to apply as a gradient
-     * @returns {string|CanvasGradient} Returns a string fillstyle value, or a canvas gradient
+     * @returns {string|CanvasGradient|HTMLImageElement} Returns a string
```

Comment:
I think HTMLImageElement can be type hinted as CanvasImageSource instead because createPattern allows more than just <img> tags.

https://developer.mozilla.org/en-US/docs/Web/API/CanvasImageSource

Reply:
Agreed, updated and elaborated the method's JSDoc.  Please review, thanks